aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -110,6 +111,7 @@ public GlueCatalog() {
 
   @Override
   public void initialize(String name, Map<String, String> properties) {
+    this.catalogProperties = properties;
```

Comment:
Nit: Might want to wrapt his in `ImmutableMap.copyOf(properties)` (which is a no-op if the underlying map is an `ImmutableMap`. Though again if you need null-values then that won't work for you.